src/server/game/Spells/SpellEffects.cpp

```diff
@@ -4392,12 +4392,15 @@ void Spell::EffectPullTowards(SpellEffIndex effIndex)
     if (!unitTarget)
         return;
 
-    float speedXY = m_spellInfo->Effects[effIndex].MiscValue / 10.0f;
-    float speedZ = damage / 10.0f;
-    Position pos;
-    pos.Relocate(m_caster);
+    Position pos = m_caster->GetFirstCollisionPosition(m_caster->GetCombatReach(), m_caster->GetRelativeAngle(unitTarget));
+
+    // This is a blizzlike mistake: this should be 2D distance according to projectile motion formulas, but Blizzard erroneously used 3D distance
```

Comment:
Maybe make a small change like adding a period dot ( . ) at the end of the comment (unless you know anything else that could need an update).